README.rst

```diff
@@ -77,65 +61,44 @@ following packages installed:
 +-------------+--------------------+
-| numpy       | 1.13 or higher     |
-+-------------+--------------------+
-| scipy       | 1.4.0 or higher    |
-+-------------+--------------------+
-| ASE         | 3.18 or higher     |
-+-------------+--------------------+
 
-Other necessary packages (such as Eigen and PyBind11) are downloaded
-automatically when compiling Rascal.
+Other necessary packages Eigen, PyBind11 and wigxjpf are downloaded
+automatically when compiling librascal.
+
 
-The following packages are required for some optional features:
+Installation
+------------
+
+.. start-install
+
+When the dependencies are met, the python package can be installed with:
+
+.. code:: bash
+
+   pip install librascal
+
+For optional features of librascal the following python packages are required:
 
 +--------------------------+-------------+--------------------+
-| Feature                  | Package     | Required version   |
+| Librascal feature        | Package     | Required version   |
 +==========================+=============+====================+
 | Feature compression      | skcosmo     | 0.1.0 or later     |
 +--------------------------+-------------+--------------------+
 | Rotational algebra       | sympy       | 1.4 or later       |
 | (Clebsch-Gordan coeffs.) |             |                    |
 +--------------------------+-------------+--------------------+
-| Building documentation   | pandoc      | (latest)           |
-+--------------------------+-------------+--------------------+
-|                          | sphinx      | 2.1.2 or later     |
-+--------------------------+-------------+--------------------+
-|                          | breathe     | 4.14.1 or later    |
-+--------------------------+-------------+--------------------+
-|                          | nbsphinx    | 0.8.1 or later     |
-+--------------------------+-------------+--------------------+
-
-Compiling
-~~~~~~~~~
-
-To compile the code it is necessary to have CMake 3.0 and a C++ compiler
-supporting C++14. During the configuration, it will automatically try to
-download the external libraries on which it depends:
-
--  Eigen
--  Pybind11
--  Boost (only the unit test framework library)
--  Python3
-
-And the following libraries to build the documentation:
-
--  Doxygen
--  Sphinx
--  Breathe
 
-Beware, Python3 is mandatory. The code won’t work with a Python version
-older than 3.
-
-You can then use pip to install all python packages required for the usage
-and development of rascal:
+The dependencies for the optional features and the `introductive examples <https://cosmo-epfl.github.io/librascal/examples/examples.html>`_ can be installed with
 
 .. code:: bash
 
-    pip install -r requirements.txt
+   pip install -r requirements/common.txt
+
+Compiling
+~~~~~~~~~
 
-To configure and compile the code with the default options, on \*nix
-systems (Windows is not supported):
+To compile the code it is necessary to have CMake 3.0 and a C++ compiler
```

Comment:
I'm pretty sure we need cmake > 3.0, most likely something around 3.10 is the lowest supported version.

Reply:
updated to 3.1 (I think it is 3.1 too, my version is 3.10.2). Moved this part to the `Dependencies` section, so we have the version number only in one place.